src/SiteLink.php

```diff
@@ -18,10 +18,18 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Michał Łazowik
+ * @author Thiemo Mättig
```

Comment:
Please try to do the same.

"This change is rather unrelated no?" <- This is closed question. Like you are stating the fact that this is unrelated and asking why I'm doing it nevertheless.

"Are you adding your name as author because you are adding some docs for private code?" <- Again, a closed question. Like the only possibility you can think of is that I'm adding myself because of some private docs. Please ask open questions if you want to know what I think. Such closed questions put me in a corner, where I can't do much but fight.

I know I'm making the exact same mistake all the time. Please make me aware if I do that.